typescript/sdk/src/hook/contracts.ts

```diff
@@ -1,23 +1,30 @@
 import {
   MerkleTreeHook__factory,
   StaticMerkleRootMultisigIsm__factory,
+  StaticMessageIdMultisigIsm__factory,
 } from '@hyperlane-xyz/core';
 
 export const merkleRootHookFactories = {
   hook: new MerkleTreeHook__factory(),
 };
 
+export const messageIdMultisigIsmFactory = {
+  ism: new StaticMessageIdMultisigIsm__factory(),
+};
```

Comment:
this still only works for message ID?
why do we need to redefine these factories at all?

Reply:
not needed for single hyperlaneHookDeployer - https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2747/commits/a53dd91d02cc382612463742e19dcb8b278d1a73